libs/custom-paper-handler/src/driver/scanner_config.ts

```diff
@@ -9,7 +9,8 @@ export type PaperMovementAfterScan =
 export type ScanLight = 'red' | 'green' | 'blue' | 'white';
 export type ScanDataFormat = 'BW' | 'grayscale';
 export type Resolution = 100 | 150 | 200 | 250 | 300;
-export type ScanDirection = 'forward' | 'backward' | 'in_park';
+export const scanDirections = ['forward', 'backward', 'in_park'] as const;
```

Comment:
Allows printing of all valid scan directions in the CLI here: https://github.com/votingworks/vxsuite/pull/4993/files#diff-9b400756f5e8f9849559bce70d1e273e357200c02fea2d5e808e58e33f702ad4R133